py_zipkin/encoding/_decoders.py

```diff
@@ -1,39 +1,22 @@
 import logging
-import socket
-import struct
-from typing import Dict
 from typing import List
-from typing import Optional
-from typing import Tuple
 
-from thriftpy2.protocol.binary import read_list_begin
-from thriftpy2.protocol.binary import TBinaryProtocol
-from thriftpy2.thrift import TType
-from thriftpy2.transport import TMemoryBuffer
-
-from py_zipkin.encoding._helpers import Endpoint
 from py_zipkin.encoding._helpers import Span
 from py_zipkin.encoding._types import Encoding
-from py_zipkin.encoding._types import Kind
 from py_zipkin.exception import ZipkinError
-from py_zipkin.thrift import zipkinCore
-
-_HEX_DIGITS = "0123456789abcdef"
-_DROP_ANNOTATIONS = {"cs", "sr", "ss", "cr"}
 
 log = logging.getLogger("py_zipkin.encoding")
 
 
 def get_decoder(encoding: Encoding) -> "IDecoder":
     """Creates encoder object for the given encoding.
-
     :param encoding: desired output encoding protocol
     :type encoding: Encoding
     :return: corresponding IDecoder object
     :rtype: IDecoder
     """
     if encoding == Encoding.V1_THRIFT:
-        return _V1ThriftDecoder()
+        raise NotImplementedError(f"{encoding} decoding not yet implemented")
```

Comment:
nit: should we call this "decoding no longer supported" to be consistent with the message in the encoder